### DISCUSSION TITLE
Java version

### DISCUSSION THREAD

**Discussion by martin617github** [Axway-API-Management-Plus/apim-cli#491] [Q&A] [answered]
Hello,

Discovered that my Java version was too old to run apim-cli 1.14.4 (I was still running jdk1.8.0_202, for some legacy app reasons).
It started working for me when I used Java 17, which I also had already installed.
Could not find a required Java version in the man page of apim-cli. Did I overlook this piece of info?

Greetings.

**Comment by rathnapandi** [comment, marked as answer]
I will add it to this documentation.
